go-apps/meepctl/utils/kibana.go

```
@@ -112,9 +117,9 @@ func isKibanaReady(cobraCmd *cobra.Command) bool {
 }
```

Error in isKibanaReady is never reported.
Try to adopt the
if err != nil {
do something
}
else do something else

structure from now on

---

Error is not reported because it is only used to trigger a Boolean response. The error already got processed in the function uploadDefaultIndex called by the isKibanaReady function. That was my rationale for not duplicating reporting the error.

---

OK - then return type of uploadDefaultIndex needs to be boolean since the error is useless

---

fixed as part of my latest commit